dictionaries/software-terms/src/software-terms.txt

```diff
@@ -1704,3 +1704,9 @@ zip
 zlib
 ZPool
 ZPools
+Spirent
```

Comment:
These look more like tools than terms.

Reply:
This time it's really a company https://www.spirent.com/

Reply:
@digger-yu,

Would you please put companies in the companies file and the tools in the tools file?

Reply:
done